src/node_v8.cc

```diff
@@ -498,6 +699,7 @@ void RegisterExternalReferences(ExternalReferenceRegistry* registry) {
   registry->Register(GCProfiler::New);
   registry->Register(GCProfiler::Start);
   registry->Register(GCProfiler::Stop);
+  registry->Register(getCppHeapStatistics);
```

Comment:
Just a tiny style suggestion, but as you can see here, the convention is mostly that the C++ functions are named in UpperCamelCase